web/themes/custom/cloudy/templates/search/facets-item-list.html.twig

```diff
@@ -47,6 +47,6 @@
   {%- endif %}
 
 {% if facet.widget.type == "dropdown" %}
-  <label id="facet_{{ facet.id }}_label">Facet {{ facet.label }}</label>
+  <label id="facet_{{ facet.id }}_label">{{ 'facet'|t|capitalize }} {{ facet.label }}</label>
```

Comment:
I wonder if we want this to be the word "filter" instead of "facet" for better usability. It probably doesn't matter at the moment as we don't use the dropdown facet type for anything, but just asking.